spec/order_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'order'
+
+describe Order do
+  let(:menu_klass){ double('menu_klass',new: menu) }
+  subject(:order){ described_class.new(menu_klass.new) }
+  let(:dish1){ double('dish') }
+  let(:dish2){ double('dish') }
+  # let(:dish1){ double('dish', name: 'dish1', price: 7.5) }
+  # let(:dish2){ double('dish', name: 'dish2', price: 10.0) }
+  let(:menu){ double('menu', dishes: { dish1 => 7.5, dish2 => 10.0 })}
+
+
+
+  describe '#initialize' do
+    it 'creates an empty order' do
+      expect(order.overview.empty?).to be true
+    end
+
+    it 'initiates menu' do
+      expect(order.menu).to eq menu
+    end
+  end
+
+  describe '#show_menu' do
+    it 'shows the menu' do
+      expect(order.show_menu).to eq menu
+    end
+  end
+
+  describe '#select_dishe' do
+    it 'insert a single dish with 1 in quantity into the order' do
+      order.select_dish(1,dish1)
+      expect(order.overview[0]).to include 1, dish1, 7.5
+    end
+
+    it 'insert multiple dish with 1 in quantity into the order' do
+      quantity = 1
+      order.select_dish(quantity,dish1)
+      order.select_dish(quantity,dish2)
+      expect(order.overview).to include [1, dish1, menu.dishes[dish1]],[quantity,dish2,menu.dishes[dish2]]
```

Comment:
Metrics/LineLength: Line is too long. [106/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)